src/middlewared/middlewared/api/v25_04_0/static_route.py

```diff
@@ -0,0 +1,46 @@
+from middlewared.api.base import BaseModel, Excluded, excluded_field
+
+from pydantic import IPvAnyAddress
+
+
+__all__ = [
+    "StaticRouteEntry",
+    "StaticRouteUpdateArgs",
+    "StaticRouteUpdateResult",
+    "StaticRouteCreateArgs",
+    "StaticRouteCreateResult",
+    "StaticRouteDeleteArgs",
+    "StaticRouteDeleteResult",
+]
+
+
+class StaticRouteEntry(BaseModel):
+    destination: IPvAnyAddress
+    gateway: IPvAnyAddress
+    destription: str = ""
+    id: int
+
+
+class StaticRouteUpdateArgs(BaseModel):
+    id: Excluded = excluded_field()
```

Comment:
I believe the fields here should exactly match the parameters of the endpoint, so `id` --> `id_`